string name
int32 machine_num
int32 start
int32 finish
int32 duration